src/e2e-test/java/io/cdap/plugin/bigquerymultitable/stepdesign/MultiTableClient.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright © 2023 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package io.cdap.plugin.bigquerymultitable.stepdesign;
+
+import io.cdap.e2e.utils.ConstantsUtil;
+import io.cdap.e2e.utils.PluginPropertyUtils;
+
+import java.io.IOException;
+import java.sql.Connection;
+import java.sql.SQLException;
+import java.sql.Statement;
+
+import static io.cdap.plugin.bigquerymultitable.stepdesign.BQMultiTableValidation.getMysqlConnection;
```

Comment:
nit: avoid using static imports, it makes the code unreadable.

Reply:
Resolved